lua/cinema/scoreboard/cl_playerlist.lua

```diff
@@ -64,7 +86,7 @@ function UpdateMutes()
     end
 end
 
-timer.Create("updatemutes", 1, 0, UpdateMutes)
+timer.Create("updatemutes", 1, 0, UpdateMuteHistory)
```

Comment:
So you're gonna read and write back a file every second, which synchronously hangs the client? Please come up with a better solution

Reply:
yea I didn't know what the impact was and I thought it was fine since the timer iterated through every player in the server before